package.json

```diff
@@ -49,8 +49,7 @@
     "@types/js-cookie": "^3.0.2",
     "@types/lodash": "^4.14.180",
     "@types/lodash-es": "^4.17.7",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
+    "@types/react-dom": "^18.0.11",
```

Comment:
the reason behind why I removed @types/react is here:
https://stackoverflow.com/a/66556011/7091035

Building works perfectly fine now.